CosmosDBDotnetTemplates.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageType>Template</PackageType>
+    <PackageVersion>1.0.0</PackageVersion>
+    <PackageId>Microsoft.Azure.Cosmos.Templates</PackageId>
+    <Title>Cosmos DB Dotnet Templates</Title>
+    <Authors>Cosmos DB</Authors>
+    <Description>Dotnet templates for Cosmos DB.</Description>
+    <PackageTags>Cosmos-DB;WebAPI,API-for-MongoDB</PackageTags>
+
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
.NET 5 is not an LTS version and will be deprecated 6 months after .NET6 is released. Should it target a LTS version?
https://dotnet.microsoft.com/platform/support/policy/dotnet-core

Reply:
Good point, I'll change this to 3.1

Reply:
I couldn't change this to 3.1 because I need to use `JsonIgnoreCondition` which is only available in 5. Looks like end of support for both of these versions is only a few months apart so hoping this is okay.